Samples/android/Kotlin/Android-SDK-Demo-App/app-v7-java/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
+        android:networkSecurityConfig="@xml/network_security_config"
+        android:theme="@style/Theme.AndroidSDKDemoApp">
+        <activity
```

Comment:
Suggestion to add `configChanges` to avoid screen orientation issues:
`android:configChanges="keyboardHidden|orientation|screenSize|screenLayout|smallestScreenSize|uiMode"`